packages/lodestar/src/chain/lightClient/proofs.ts

```diff
@@ -1,9 +1,9 @@
-import {allForks} from "@chainsafe/lodestar-types";
+import {altair} from "@chainsafe/lodestar-types";
 import {TreeBacked} from "@chainsafe/ssz";
 import {FINALIZED_ROOT_GINDEX} from "@chainsafe/lodestar-params";
 import {SyncCommitteeWitness} from "./types";
 
-export function getSyncCommitteesWitness(state: TreeBacked<allForks.BeaconState>): SyncCommitteeWitness {
+export function getSyncCommitteesWitness(state: TreeBacked<altair.BeaconState>): SyncCommitteeWitness {
```

Comment:
shouldn't this stay `allForks` to allow for bellatrix and future forks?

Reply:
allForks doesn't have the fields for sync committees and this doesn't compile. The correct type would be allForks except phase0, but with current allforks types that's not possible. With SSZ refactor I can do that since allForks is purely a union